temporal-sdk/src/main/java/io/temporal/internal/sync/ActivityExecutionContextImpl.java

```diff
@@ -29,12 +29,7 @@
 import io.temporal.api.common.v1.Payloads;
 import io.temporal.api.workflowservice.v1.RecordActivityTaskHeartbeatRequest;
 import io.temporal.api.workflowservice.v1.RecordActivityTaskHeartbeatResponse;
-import io.temporal.client.ActivityCanceledException;
-import io.temporal.client.ActivityCompletionClient;
-import io.temporal.client.ActivityCompletionException;
-import io.temporal.client.ActivityCompletionFailureException;
-import io.temporal.client.ActivityNotExistsException;
-import io.temporal.client.ActivityWorkerShutdownException;
+import io.temporal.client.*;
```

Comment:
Undo. Change your IDE settings to avoid such rollups.

Reply:
Ah, I've switched from Community to Ultimate and reset my settings. Fixed.